lib/sales_engine.rb

```diff
@@ -0,0 +1,12 @@
+require 'csv'
+require './lib/merchant_repository'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.